sia/src/transactions.rs

```diff
@@ -935,4 +1301,407 @@ mod tests {
         let json_str = "{\"siacoinInputs\":[],\"siacoinOutputs\":[],\"fileContracts\":[],\"fileContractRevisions\":[],\"storageProofs\":[],\"siafundInputs\":[],\"siafundOutputs\":[],\"minerFees\":[],\"arbitraryData\":[],\"signatures\":[]}";
         test_serialize_json(&transaction, json_str);
     }
+
+    #[test]
+    fn test_serialize_siacoin_element() {
```

Comment:
This test would be slightly better if it wasn't exclusively 0 values. To make sure we test the order of the fields within the encoding and endianness.